src/components/tests-to-run-panel/tests-to-run-panel.tsx

```diff
@@ -105,8 +105,8 @@ export const TestsToRunPanel = () => {
             />
           </div>
           <div tw="text-14 mt-4 px-6 space-y-4">
-            {getSelectedTests().map(({ id = "", details }) => (
-              <div tw="flex items-center gap-x-4" key={id} data-test="tests-to-run-modal:tests-list:test">
+            {getSelectedTests().map(({ details }) => (
+              <div tw="flex items-center gap-x-4" data-test="tests-to-run-modal:tests-list:test">
```

Comment:
if backend send ununiq id for you, it in not our problem

Reply:
Just remove key?